Common/Transforms/itkEulerTransform.h

```diff
@@ -46,7 +46,7 @@ class EulerGroup
  */
 
 template <>
-class EulerGroup<2>
+class ITK_TEMPLATE_EXPORT EulerGroup<2>
```

Comment:
I wonder if `ITK_TEMPLATE_EXPORT` should also be applied to explicit (full) template specializations, like `EulerGroup<2>`...